metadata.rb

```diff
@@ -6,12 +6,12 @@
 license          'Apache-2.0'
 description      'Installs/Configures Splunk Servers and Forwarders'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.38.0'
+version          '2.39.0'
 
 source_url       'https://github.com/cerner/cerner_splunk'
 issues_url       'https://github.com/cerner/cerner_splunk/issues'
 
-chef_version     '>= 12.7', '< 16'
+chef_version     '>= 12.7'
```

Comment:
I see your concern. We discussed about this. The reason we decided to remove the upper limit is that having to uplift every cookbook even though sometimes no breaking changes are introduce to the consumer cookbook. We decide to advertise supported(tested) versions in Readme. This way anyone who want to try this out with >=17 can still do it and if there are breaking changes it can be addressed and tested with new major version bump on the cookbook. 

For this cookbook, if you have strong preference, I can add upper limit for `<17`.

Reply:
I prefer to keep the upper bound to make it explicit in code instead of just in a readme.  @som3guy @srikanthreddy5 I would be interested to hear your opinions here.

Reply:
I like the upper bound as well to at least manage expectations.